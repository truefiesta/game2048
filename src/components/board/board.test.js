import React from "react";
import renderer from "react-test-renderer";
import Board from "./board";

describe(`Board snapshot`, () => {
    it(`renders`, () => {
        const tileValues = [0, 0, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 2, 2];
        const board = renderer.create(
            <Board
                tileValues={tileValues}
            />
        )
        .toJSON();

        expect(board).toMatchSnapshot();
    });
});