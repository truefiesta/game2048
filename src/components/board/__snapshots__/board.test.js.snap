// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board snapshot renders 1`] = `
<section
  className="board board-4 box"
>
  <div
    className="tile tile-0"
  >
    <span>
      0
    </span>
  </div>
  <div
    className="tile tile-0"
  >
    <span>
      0
    </span>
  </div>
  <div
    className="tile tile-2"
  >
    <span>
      2
    </span>
  </div>
  <div
    className="tile tile-4"
  >
    <span>
      4
    </span>
  </div>
  <div
    className="tile tile-8"
  >
    <span>
      8
    </span>
  </div>
  <div
    className="tile tile-16"
  >
    <span>
      16
    </span>
  </div>
  <div
    className="tile tile-32"
  >
    <span>
      32
    </span>
  </div>
  <div
    className="tile tile-64"
  >
    <span>
      64
    </span>
  </div>
  <div
    className="tile tile-128"
  >
    <span>
      128
    </span>
  </div>
  <div
    className="tile tile-256"
  >
    <span>
      256
    </span>
  </div>
  <div
    className="tile tile-512"
  >
    <span>
      512
    </span>
  </div>
  <div
    className="tile tile-1024"
  >
    <span>
      1024
    </span>
  </div>
  <div
    className="tile tile-2048"
  >
    <span>
      2048
    </span>
  </div>
  <div
    className="tile tile-4096"
  >
    <span>
      4096
    </span>
  </div>
  <div
    className="tile tile-2"
  >
    <span>
      2
    </span>
  </div>
  <div
    className="tile tile-2"
  >
    <span>
      2
    </span>
  </div>
</section>
`;
